### PR TITLE
Create Newtons_law_of_gravitation.md

### PR DESCRIPTION
**Pull request by KaranSinghS**
Explanation of the newtons law of gravitation with the parameters being defined and with the explanation on how to use and the 
restrictions and assumptions made

### CODE REVIEW

File: en/Newtons_law_of_gravitation.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Let M_1 be a Iron ball of mass `5 * 10^6 Kg` and M_2 be a Copper ball of mass `10^4 Kg` and the seperation between the 2 objects **r**=100m
+Then the Graviational force between the 2 distinct objects is = `0.0003337 N`  or  `3.337 * 10^-4 N` 
+
+# Source
+- [Wikipedia](https://en.wikipedia.org/wiki/Newton%27s_law_of_universal_gravitation)
+- [Youtube](https://www.youtube.com/watch?v=ruBfXIVSYZ8)
+
+# Implementation
+- [newtos_law_of_gravitations](https://github.com/TheAlgorithms/Algorithms-Explanation/blob/master/en/Basic%20Math/Arithmetic%20Progressions.md)
```
Comment (Panquesito7) [inline en/Newtons_law_of_gravitation.md:50]: The implementations header is used if there are any implementations of this algorithm in some of [The Algorithms repositories](https://github.com/orgs/TheAlgorithms/repositories). If there are none, feel free to remove this part.

File: en/Newtons_law_of_gravitation.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+This law is used to determine the Force of attraction between 2 distinct Objects/Bodies , in order to calculate the rate of acceleration of either body towards the 
+other body with one being at a complete stop .
+
+# Example:-
+
+Let M_1 be a Iron ball of mass `5 * 10^6 Kg` and M_2 be a Copper ball of mass `10^4 Kg` and the seperation between the 2 objects **r**=100m
+Then the Graviational force between the 2 distinct objects is = `0.0003337 N`  or  `3.337 * 10^-4 N` 
+
+# Source
```
Comment (Panquesito7) [inline en/Newtons_law_of_gravitation.md:45]:
```suggestion
# Sources

```

File: en/Newtons_law_of_gravitation.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+- [Wikipedia](https://en.wikipedia.org/wiki/Newton%27s_law_of_universal_gravitation)
+- [Youtube](https://www.youtube.com/watch?v=ruBfXIVSYZ8)
```
Comment (Panquesito7) [inline en/Newtons_law_of_gravitation.md:47]:
```suggestion
- [YouTube video](https://www.youtube.com/watch?v=ruBfXIVSYZ8)
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Hey there! Thanks for contributing. 🎉
Please move your file to its appropriate directory.

File: en/Newtons_law_of_gravitation.md (hunk 1 above)
Comment (KaranSinghS) [reply en/Newtons_law_of_gravitation.md]: I have made the suggested changes, would it now be possible to accept this

**Pull request closed without merging**